File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do{
        height = get_int("height: ");
    }
    while (!(height >= 1 && height <= 8));

    for (int i = 1; i <= height; i++) {

        /*for (int k = 1; k <= height - i; k++) {
            printf(" ");
        }
        for (int j = 1; j <= i; j++) {
            printf("#");
        }
        printf("  ");*/
        for (int k = 1; k <= i; k++) {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario/less/mario.py
from cs50 import get_int

def print_pyramid(height):
    for i in range(height):
        spaces = " " * (height - i - 1)
        hashes = "#" * (i + 1)
        print(spaces + hashes)

def main():
    while True:
        height = get_int("Height: ")
        if height <= 8 and != 0:
            break  # Sai do loop se a entrada for válida
        else:
            print_pyramid(height)

main()